spec/models/open_graph_cache_spec.rb

```diff
@@ -57,5 +57,21 @@
         expect(ogc.video_url).to eq("https://bandcamp.com/EmbeddedPlayer/v=2/track=12/size=small")
       end
     end
+
+    context "a mixed case hostname" do
+      it "downcases the hostname" do
+        stub_request(:head, "http:///wetter.com").with(headers: {
+          'Accept'=>'text/html',
+          'User-Agent'=>'OpenGraphReader/0.6.2 (+https://github.com/jhass/open_graph_reader)'
+        }).to_return(status: 200, body: "", headers: {
```

Comment:
Layout/IndentHash: Indent the right brace the same as the first position after the preceding left parenthesis.